packages/tools/pluggable-widgets-tools/configs/eslint.ts.base.json

```diff
@@ -144,15 +144,7 @@
     },
     "overrides": [
         {
-            "files": [
-                "webpack.config.dev.js",
-                "webpack.config.prod.js",
-                "prettier.config.js",
-                "*.webmodeler.*",
-                "*.editorPreview.*",
-                ".eslintrc.js",
-                "wdio.conf.js"
-            ],
+            "files": ["prettier.config.js", "*.webmodeler.*", "*.editorPreview.*", ".eslintrc.js", "wdio.conf.js"],
```

Comment:
should we add rollup custom config here?

Reply:
Not sure why should we disable `no-var-requires` or `class-name-casing` for it. Moreover, it is going to be in js anyway => the whole `@typescript-eslint` does not apply

Reply:
Cool 👍